dist/interfaces.js

```diff
@@ -0,0 +1,2 @@
+"use strict";
+Object.defineProperty(exports, "__esModule", { value: true });
```

Comment:
This looks odd, I mean there are no interfaces in JS and I guess its fine, but this is just an empty file

Reply:
Huh, that is strange. I'm not sure what's happening here, but things still work

Reply:
This is the file that only holds the interface, interfaces don't exist in JS (only TS) which is why the transpiled file is basically empty

Reply:
Oh right, so it shouldn't add it in the first place